conda_forge_tick/auto_tick.py

```diff
@@ -670,6 +670,7 @@ def migrator_status(
     build_sequence = list(cyclic_topological_sort(gx2, top_level))
 
     feedstock_metadata = dict()
+    feedstock_ctx = FeedstockContext
```

Comment:
Right but this is a python Class, so to instantiate you need to call its init method etc. See eg https://python101.pythonlibrary.org/chapter11_classes.html

Reply:
yeah sorry i am getting the hang of it.